core/cmd/local_client.go

```diff
@@ -418,13 +364,13 @@ func (cli *Client) RebroadcastTransactions(c *clipkg.Context) (err error) {
 	}
 
 	lggr := cli.Logger.Named("RebroadcastTransactions")
-	db, err := openDB(cli.Config, lggr)
-	if err != nil {
-		return cli.errorOut(errors.Wrap(err, "opening db"))
+	ldb := pg.NewLockedDB(cli.Config, lggr)
+	if err = ldb.Open(context.Background()); err != nil {
```

Comment:
Do we want this context to be linked to a sigterm?

Reply:
Not relevant anymore, given the new API.